### PR TITLE
Add Circe Stable

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

**Comment by naftulikay** [comment]
@danirabbit thank you so much for creating a Docker image! this is extremely helpful for automation work :pray:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,7 +28,6 @@ jobs:
           stable,
           stable-installer,
           ubuntu-mantic-unstable,
```
Comment (ryonakano) [inline .github/workflows/ci.yml]: Maybe we can remove this too?

Comment (ryonakano) [reply .github/workflows/ci.yml]: Ah ok, I saw #45

**Review by ryonakano — APPROVED** [review]
Wow, new codename is here :classical_building:

**Pull request merged**